docker-compose.yml

```diff
@@ -10,6 +10,7 @@ services:
       - mongodb
       - mysql
       - postgres
+      - presto
```

Comment:
https://github.com/DataDog/dd-trace-rb/pull/975 seems to address this, I think a rebase will like address the changes in this file.